espanol/pesadilla/pesadilla.md

```diff
@@ -1 +1,4 @@
-¿Real o no real?
\ No newline at end of file
+¿Real o no real?
+
+[De repente, entro un perro](https://github.com/udacity/create-your-own-adventure/blob/master/espanol/buba/buba.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.